t/apicast-async-reporting.t

```diff
@@ -129,8 +129,13 @@ GET /test?user_key=foo
 --- response_body
 api response
 --- error_code: 200
---- error_log env
-backend client uri: https://127.0.0.1:$TEST_NGINX_RANDOM_PORT/transactions/authrep.xml?service_token=token-value&service_id=42&usage%5Bhits%5D=1&user_key=foo ok: true status: 200
+--- error_log env eval
+[
+  qr/backend client uri\: https\:\/\/127\.0\.0\.1:$TEST_NGINX_RANDOM_PORT\/transactions\/authrep.xml\?.*?(service_id=42).*? ok\: true status\: 200/,
```

Comment:
Would it be better to check the params in a `location`? Similar to what we have in other tests modified in this PR: https://github.com/3scale/APIcast/pull/1272/commits/4779efe9f29544aa58ad858d38d0d93c418cbde8#diff-14b2ba18db23d9d85a7017bca7bf61564525518b3f6305633788656396c1757fR614